src/annotator/config/index.js

```diff
@@ -8,7 +10,12 @@ import { toBoolean } from '../../shared/type-coercions';
  *
  * @typedef ConfigDefinition
  * @prop {(settings: SettingsGetters) => any} getValue -
- *   Method to retrieve the value from the incoming source
+ *  Method to retrieve the value from the incoming source
+ * @prop {boolean} allowInBrowserExt -
+ *  Allow this name to be available in the browser extension. If this is false
+ *  and browser extension context is true, use `defaultValue` if provided otherwise
+ *  ignore the config key
+ * @prop {any} [defaultValue] - Sets a default if `valueFn` returns undefined
```

Comment:
```suggestion
 * @prop {any} [defaultValue] - Sets a default if `getValue` returns undefined
```

The `valueFn` property was renamed after an earlier discussion. This needs to be updated accordingly.